Automattic/jetpack
# VideoPress: reload block on rating change

Fixes Automattic/greenhouse#1005

#### Changes proposed in this Pull Request:

When updating the rating on a VideoPress video block, the block is now reloaded to reflect the changes (for the age gate in particular).
This PR also fixes the meta API that was previously checking the existing rating instead of the new one.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a Jetpack site with VideoPress enabled
* Edit or create a post with a /video block
* Upload or select a video from your media library already uploaded on VideoPress
* In the block settings, play with the "Rating" option
![Capture d’écran 2021-11-05 à 11 51 17](https://user-images.githubusercontent.com/1420594/140499628-d84aae21-e185-41b3-98fd-d234d73da1aa.png)
* ✅ The block should be regenerated every time the "rating" option is changed
* ✅ The block should display the age gate if "R" is selected
* ✅ The block should not display the age gate if anything else is selected
* After selecting a rating, reload the page and select the video block
* ✅ The previously selected rating should be displayed
